### PR TITLE
Main Table Component

### PR DESCRIPTION
**Pull request by nil-amrutlal-dept**
### Modify this title

Related Issue/Asana ticket: \_

Short description: \_
image

### Test scenarios

- description of the (manually) executed test scenarios

### Reminders (if applicable)

- [x] I ran `pnpm install` and `pnpm test` in the root of the monorepo
      (optionally with `--filter=...package...` to exclude non-affected
      projects)
- [x] I ran `pnpm changeset` in the root of the monorepo
- [x] Test coverage has not decreased
- [ ] Docs have been updated to reflect changes in PR (don't forget
      docs.kadena.io)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206590017183452

### CODE REVIEW

File: packages/apps/explorer/src/components/block-table/block-header/block-header.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+            <Text variant="body" size="small" bold>
+              {height}
+            </Text>
+          </Stack>
+        ))}
+      </Stack>
+      {!isCompact && (
+        <Stack
+          flexDirection={'column'}
```
Comment (sstraatemans) [inline packages/apps/explorer/src/components/block-table/block-header/block-header.tsx:89]: these can go without {} ?

File: packages/apps/explorer/src/services/format.ts [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+/**
+ * this function gets your numeric value and returns a string with the value and the unit (e.g. 1023 will be formatted to 1.23 K)
+ * @param number Value to be formatted
+ * @returns string value with unit
+ *
+ */
+export function formatNumberWithUnit(number: number): string {
   if (number === 0) {
-    return `0 ${unit !== undefined ? unit : ''}`;
+    return `0`;
```
Comment (sstraatemans) [inline packages/apps/explorer/src/services/format.ts:11]: it returns undefineds again
image

File: packages/apps/explorer/src/constants/search.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import type { ISearchItem } from '@/components/search/search';
+import type { ISearchItem } from '@/components/search-component/search-component';
```
Comment (sstraatemans) [inline packages/apps/explorer/src/constants/search.ts:1]: maybe make it '@/components/search/search-component/search-component'
and have all the search components close to eachother?

im adding '@/components/search/search-results/search-results'

File: packages/apps/explorer/src/services/format.ts (hunk 2 above)
Comment (nil-amrutlal-dept) [reply packages/apps/explorer/src/services/format.ts:10]: Good catch! The issue here was with the unit. Fixed it

**Review by sstraatemans — APPROVED** [review]

**Pull request merged**